tests/gis_tests/distapp/tests.py

```diff
@@ -517,9 +528,7 @@ def test_distance_projected(self):
             dist_qs = [dist1]
         else:
             dist2 = SouthTexasCityFt.objects.annotate(distance=Distance('point', lagrange)).order_by('id')
-            # Using EWKT string parameter.
-            dist3 = SouthTexasCityFt.objects.annotate(distance=Distance('point', lagrange.ewkt)).order_by('id')
```

Comment:
https://docs.djangoproject.com/en/1.8/ref/models/expressions/#django.db.models.Func

> Positional arguments can be expressions or Python values. **Strings are assumed to be column references and will be wrapped in F() expressions** while other values will be wrapped in Value() expressions.

Do you still think we should handle geo_input parameters?